src/ManifoldsExtentions.jl

```diff
@@ -53,16 +77,16 @@ end
 
 #FIXME fix back to SA
 function getPointIdentity(G::SpecialOrthogonal{N},::Type{T}=Float64) where {N,T<:Real}
-  # return SMatrix{N,N, T}(I)
-  return Matrix{T}(I, N, N)
+  return SMatrix{N,N, T}(I)
+  # return Matrix{T}(I, N, N)
```

Comment:
Note to self: "Myself, remember to switch these back"

Reply:
did you remember?